README.md

```diff
@@ -378,59 +378,65 @@ The final step once your app is setup is to list in the HubSpot marketplace. You
 </details>
 
 <details>
  <summary> Can users use multiple integrations at the same time? </summary>
  <p>
-    Yes, users can use multiple 3rd party calling integrations at the same time. They can use the provider switcher presented after clicking on the call button to seamlessly switch between providers.
+    Yes, users can use multiple 3rd party calling integrations at the same time. They can use the provider switcher presented after clicking on the call button to seamlessly switch between providers. </p>
 </details>
 
 <details>
  <summary> Can free users install app integrations? </summary>
  <p>
-    No, this feature is available on to paid customers. At this point in time, users will be able to install the app but will not be able to access it.
+    No, this feature is available on to paid customers. At this point in time, users will be able to install the app but will not be able to access it. </p>
 </details>
 
 <details>
  <summary> How can users install the integration? </summary>
  <p>
-    The current SDK does not support inbound calling. We understand this is a big feature request and will have more to share about our plans for inbound calling in the future. 
+    The current SDK does not support inbound calling. We understand this is a big feature request and will have more to share about our plans for inbound calling in the future. </p>
 </details>
 
 <details>
  <summary> If a user already has my app installed, does the integration automatically show up? </summary>
  <p>
-    Yes, if a user already has installed your app, and you are updating the same app with the calling extensions the integration will automatically show up. Currently, there is no way for the developer to enable the call widget only to a subset of customers.
+    Yes, if a user already has installed your app, and you are updating the same app with the calling extensions the integration will automatically show up. Currently, there is no way for the developer to enable the call widget only to a subset of customers.</p>
 </details>
 
 <details>
  <summary> Can any user install or uninstall an app? </summary>
  <p>
-    No. Only users who have necessary permissions can install and uninstall an app. These permissions can be found in HubSpot portal settings page in the "Users & Teams" tab.
+    No. Only users who have necessary permissions can install and uninstall an app. These permissions can be found in HubSpot portal settings page in the "Users & Teams" tab. </p>
+</details>
+
+<details>
+ <summary> Need help with the SDK? </summary>
+ <p>
+    Head over to the HubSpot developer support forum here (link https://community.hubspot.com/t5/APIs-Integrations/bd-p/integrations) and post your questions. Someone from our developer support team will answer your queries. </p>
```

Comment:
did you want to do [HubSpot developer support forum](link https://community.hubspot.com/t5/APIs-Integrations/bd-p/integrations) ?